source/yats/include/yats/task_configurator.h

```diff
@@ -57,14 +58,22 @@ class task_configurator : public abstract_task_configurator
         return find<typename helper::output_tuple, std::tuple_element_t<index, type>>(m_outputs, Id);
     }
 
+    template <uint64_t Id, typename LambdaTask>
+    void add_listener(LambdaTask task)
```

Comment:
This is not a LambdaTask. Better name would be Callable: http://en.cppreference.com/w/cpp/concept/Callable